src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/step_index_pattern/step_index_pattern.html

```diff
@@ -0,0 +1,163 @@
+<div class="kuiPanel">
+  <div class="kuiPanelHeader">
+    <div class="kuiPanelHeaderSection">
+      <h2 class="kuiPanelHeader__title">
+        Step 1 of 2: Define index pattern
+      </h2>
+    </div>
+  </div>
+
+  <div class="kuiPanelBody">
+    <div>
+      <form
+        name="stepIndexPattern.indexPatternNameForm"
+        role="form"
+        ng-submit="stepIndexPattern.goToNextStep()"
+        class="kuiVerticalRhythm"
+      >
+        <div class="createIndexPatternInputContainer kuiVerticalRhythm">
+          <!-- Index pattern input -->
+          <div>
+            <label
+              for="indexPatternNameField"
+              class="kuiLabel kuiVerticalRhythmSmall"
+            >
+              Index pattern
+            </label>
+
+            <div class="kuiVerticalRhythmSmall kuiVerticalRhythm">
+              <input
+                id="indexPatternNameField"
+                class="kuiTextInput kuiTextInput--large"
+                data-test-subj="createIndexPatternNameInput"
+                ng-model="stepIndexPattern.indexPatternName"
```

Comment:
I really think this needs at least a small debounce. The loading indicator flicker is quite distracting and it results in a ton of unnecessary network communication since each keystroke results in three requests. [the ng-model-options strategy in master](https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html#L38) should do the trick without many changes necessary.